scanomatic/data_processing/phenotyper.py

```diff
@@ -1077,7 +1077,7 @@ def _calculate_phenotypes(self):
                 plate_flat_regression_strided,
             ):
                 id1 = pos_index % plate.shape[1]
-                id0 = pos_index / plate.shape[1]
+                id0 = int(pos_index / plate.shape[1])
```

Comment:
Shouldn't this rather be this?
```suggestion
                id0 = pos_index // plate.shape[1]
```

Reply:
I don't mind changing, but I don't think I understand the difference in this case. If `pos_index` and `plate.shape[1]` are `int`s, is there any?

Reply:
Hmm, yeah.... As long as neither is negative, you're right. Is `pos_index` always positive (or nil)?

I think `//` is neater, but if they are always positive it isn't important.